src/main/webapp/help-projectConfig-credentialsSlackTokenId.html

```diff
@@ -0,0 +1,4 @@
+<div>
+	<p>The integration token to be used to send notifications to Slack. This value is used in union with the Credentials to store secrets.  If both "Integration Token ID" and "Credentials Integration Token" are set, the "Credentials Integration Token" will take precedence for security reasons.</p>
```

Comment:
The verbiage here is a bit clunky. To me it seems possible that it is not obvious enough that it should be the id of the Credentials secret. ("used in union with" is misleading about where the important data is kept). We should not mislead users to put an actual token (which looks a lot like a credentials id) into this field.